commit 9acef4a998541ac024c5b9f81a1a1ee0100afdb9
Author: Kirill A. Korinsky <kirill@korins.ky>
Date:   Thu Sep 23 10:37:14 2021 +0200

    Added `-Wunknown-pragmas` as ignored pragma
    
    Old clang fails such us shipped with Yosemite because it has no idea
    about `-Walloca`.

diff --git a/src/pci_e82545.c b/src/pci_e82545.c
index 74f199f..156484e 100644
--- a/src/pci_e82545.c
+++ b/src/pci_e82545.c
@@ -1413,6 +1413,7 @@ e82545_transmit(struct e82545_softc *sc, uint16_t head, uint16_t tail,
 	/* Allocate, fill and prepend writable header vector. */
 	if (hdrlen != 0) {
 #pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunknown-pragmas"
 #pragma clang diagnostic ignored "-Walloca"
 		hdr = __builtin_alloca((size_t)(hdrlen + vlen));
 #pragma clang diagnostic push
